### ISSUE TITLE
CC0082: ComputeExpressionCodeFixProvider crashs

### ISSUE THREAD

**Issue by danferreira** [code-cracker/code-cracker#841]
# Bug

I dont know if it is a bug or only happens with my setup.

I was playing with your extension and when I open the lamp in any of the next codes:

``` csharp
FooMethod(60 * 2);
string.Format("2 Hours in minutes: {0}", 60 * 2);
```

I got:

![bug](https://cloud.githubusercontent.com/assets/6264692/18614999/ec0c4366-7d70-11e6-8d0c-728158fc7c60.png)

I ran VS with devenv.exe /log and I got:

`Não é possível converter um objeto do tipo 'Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax' no tipo 'Microsoft.CodeAnalysis.CSharp.Syntax.BinaryExpressionSyntax'.
 at CodeCracker.CSharp.Refactoring.ComputeExpressionCodeFixProvider.<ComputeExpressionAsync>d__4.MoveNext()
 --- End of stack trace from previous location where exception was thrown ---
 at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
 at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
 at CodeCracker.CSharp.Refactoring.ComputeExpressionCodeFixProvider.<RegisterCodeFixesAsync>d__3.MoveNext()
 --- End of stack trace from previous location where exception was thrown ---
 at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
 at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
 at Microsoft.CodeAnalysis.CodeFixes.CodeFixService.<>c__DisplayClass13_0.<<AppendFixesAsync>b__2>d.MoveNext()
 --- End of stack trace from previous location where exception was thrown ---
 at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
 at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
 at Microsoft.CodeAnalysis.Extensions.IExtensionManagerExtensions.<PerformFunctionAsync>d__3`1.MoveNext()`

Version: 1.0.1

**Comment by carloscds** [comment]
@danferreira Did you try on another machine ?

**Comment by danferreira** [comment]
Yes, I did.

**Comment by carloscds** [comment]
@danferreira Bug confirmed!

**Comment by giggio** [comment]
Closed on 25b0270b through #843.
